app/models/mingle/instagram/photos_fetcher.rb

```diff
@@ -10,12 +12,24 @@ def initialize(hashtag)
 
     def fetch
       valid_photos.each {|p| p.save!}
+      verifying_integrity_for_deleted_photos
```

Comment:
The specs are failing, and I believe this is why — `fetch` previously to return a list of fetched photos, but now returns an array of `true` for each deleted photo instead (the return value of `verifying_integrity_for_deleted_photos`).